src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/Winmeta.cs

```diff
@@ -63,7 +63,15 @@ public enum EventTask
         /// <summary>
         /// Undefined task
         /// </summary>
-        None = 0
+        None = 0,
+        /// <summary>
+        /// Thread pool worker thread task
+        /// </summary>
+        ThreadPoolWorkerThread = 16,
```

Comment:
I didn't even notice that they were in the shared folder until now. I can change the attributes on the `ClrThreadPoolEventSource` to use the correct integer values cast to the specific enum types. That way there will not be any changes to the public API.